ocm-container.sh

```diff
@@ -144,6 +151,16 @@ ${AWSFILEMOUNT} \
 ${SSH_AGENT_MOUNT} \
 ${OPS_UTILS_DIR_MOUNT} \
 -v ${HOME}/.ssh/sockets:/root/.ssh/sockets \
+${PORT_MAP_OPTS} \
 ${OCM_CONTAINER_LAUNCH_OPTS} \
-ocm-container:${BUILD_TAG} ${EXEC_SCRIPT}
+ocm-container:${BUILD_TAG} ${EXEC_SCRIPT})
+
+if [ "${DISABLE_CONSOLE_PORT_MAP}" != "true" ]
+then
+  echo $($CONTAINER_SUBSYS inspect $CONTAINER | jq -r '.[].NetworkSettings.Ports."9999/tcp"[].HostPort') > /tmp/portmap
+  $CONTAINER_SUBSYS cp /tmp/portmap $CONTAINER:/tmp/portmap
+  rm -f /tmp/portmap
+fi
+
+$CONTAINER_SUBSYS start $CONTAINER --attach
```

Comment:
There's something weird happening with docker with this functionality.  I think we might have to do different things depending on our container subsys here.

For docker, the `start $CONTAINER --attach` doesn't seem to respect the password fields, so when I'm typing in my bastion password it's displayed in plaintext.

However, `docker attach $CONTAINER` respects that fine, so we might just want to do something like 
```
$CONTAINER_SUBSYS start $CONTAINER
$CONTAINER_SUBSYS attach $CONTAINER
```

Reply:
Works fine with podman too, I'll swap that.